src/Ocelot/DependencyInjection/ConfigurationBuilderExtensions.cs

```diff
@@ -84,5 +84,20 @@ public static IConfigurationBuilder AddOcelot(this IConfigurationBuilder builder
 
             return builder;
         }
+
+
+        public static IConfigurationBuilder AddOcelot(
+            this IConfigurationBuilder builder, FileConfiguration fileConfiguration, IWebHostEnvironment env)
+        {
+            const string primaryConfigFile = "ocelot.json";

```

Comment:
I would recommend to move this string constant declaration to the class level:
```csharp
public static class ConfigurationBuilderExtensions
{
    private const string primaryConfigFile = "ocelot.json";

    // usage on the lines 81 and 92.
}
```

Reply:
@raman-m Moved PrimaryConfigFile & GlobalConfigFile both at class level. Please review.